module/VuFind/config/module.config.php

```diff
@@ -283,6 +283,7 @@
             'VuFind\ContentTOCPluginManager' => 'VuFind\Service\Factory::getContentTOCPluginManager',
             'VuFind\CookieManager' => 'VuFind\Service\Factory::getCookieManager',
             'VuFind\Cover\Router' => 'VuFind\Service\Factory::getCoverRouter',
+            'VuFind\CsrfValidator' => 'VuFind\Service\Factory::getCsrfValidator',
```

Comment:
For forward-compatibility with the Service Manager 3 code, I'd recommend naming this service for the class that's being created -- i.e. Zend\Validator\Csrf. We're already using that class in the login process -- should we be using this service there as well, or is there a reason to configure it differently in two different places? (I haven't reviewed that code... so apologies if this is actually a bad suggestion because I'm missing an important detail).